Authenticator/Source/TokenListViewController.swift

```diff
@@ -42,7 +42,7 @@ class TokenListViewController: UITableViewController {
     }
 
     private var displayLink: CADisplayLink?
-    private let ring: OTPProgressRing = OTPProgressRing(frame: CGRectMake(0, 0, 22, 22))
+    private let ring: OTPProgressRing = OTPProgressRing(frame: CGRect(x: 0, y: 0, width: 22, height: 22))
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 105 characters (line_length)